src/ui/views/convlist.coffee

```diff
@@ -27,13 +27,20 @@ module.exports = view (models) ->
                 name = nameofconv c
                 if viewstate.showConvThumbs or viewstate.showConvMin
                     div class: 'thumbs thumbs-'+(if ents.length>4 then '4' else ents.length), ->
+                        additional = []
                         for p, index in ents
-                            break if index >= 4
-                            entity.needEntity(p.id)
-                            drawAvatar(p.id, viewstate, entity)
-
-                        if ents.length>4
-                            div class:'moreuser', ents.length
+                            # if there are up to 4 people in the conversation
+                            #   then draw them all, otherwise, draw 3 avatars
+                            #   and then add a +X , where X is the remaining
+                            #   number of people
+                            if index < 3 ||  ents.length == 4
+                                entity.needEntity(p.id)
+                                drawAvatar(p.id, viewstate, entity)
+                            else
+                                additional.push nameof entity[p.id]
+                        if ents.length > 4
+                            div class:'moreuser', "+#{ents.length - 3}"
```

Comment:
changed this to +X instead of showing the total number of persons in a conversation

**rationale:** I was interpreting this as the people in addition to the 3 avatars already drawn. Only by looking at the code I realised it was the total number of people in the conversation

I can revert this to previous behavior if someone doesn't agree with this :)